lib/charms/haproxy/v0/haproxy_route.py

```diff
@@ -0,0 +1,543 @@
+"""TODO: Add a proper docstring here.
```

Comment:
You are missing the docs here. Also, it would be nice to unit test this file too

Reply:
Thanks, I updated the docs.